examples/segmentation_demos.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""
+Demo script for anatomical MRI segmentation with SPM.
+
+It demos segmentations on SPM single-subject auditory.
+
+@author: Mehdi RAHIM
+"""
+
+import os
+from pypreprocess.subject_data import SubjectData
+from pypreprocess.datasets import fetch_spm_auditory
+from pypreprocess.nipype_preproc_spm_utils import _do_subject_segment
+from pypreprocess.reporting.check_preprocessing import plot_segmentation
+
+
+OUTPUT_DIR = 'segmentation_demos_output'
+def _spm_auditory_factory():
```

Comment:
The name is misleading. This has nothing to do with the factory design pattern. Please use a less connoted word.